modules/sdk/project-templates/project-templates/build.gradle

```diff
@@ -174,6 +176,26 @@ uploadProjectTemplateArchives {
 	}
 }
 
+projectTemplateJarVersions {
```

Comment:
Closures in Gradle should be sorted alphabetically.  Please move `projectTemplateJarVersions` after `processTestResources`.